examples/koa/tracer.js

```diff
@@ -9,17 +9,23 @@ const { NodeTracerProvider } = require('@opentelemetry/sdk-trace-node');
 const { SimpleSpanProcessor } = require('@opentelemetry/sdk-trace-base');
 const { JaegerExporter } = require('@opentelemetry/exporter-jaeger');
 const { ZipkinExporter } = require('@opentelemetry/exporter-zipkin');
+const { Resource } = require("@opentelemetry/resources")
+const { SemanticResourceAttributes } = require("@opentelemetry/semantic-conventions")
```

Comment:
The `require`s above use a single quote, can be nice to follow the existing style: `require('@opentelemetry/resources')`

Reply:
done